README.md

```diff
@@ -1,32 +1,31 @@
-# Show Whitespace in Brackets
+# Show Whitespace
 
-[Brackets](http://brackets.io/) extension to visualize whitespace (spaces and tabs) like [Sublime Text](http://www.sublimetext.com/) does. Adds an entry called "Show Whitespace" to the View menu, the shortcut is Command-Shift-W (or Ctrl-Shift-W on Windows). The state is remembered next time you start Brackets.
+> A [Brackets](http://brackets.io/) extension to visualize whitespace (both spaces and tabs) in the same style as [Sublime Text](http://www.sublimetext.com/).
```

Comment:
adding sublime text does not really seem to add much value to... but maybe that's my biased perception :)